### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by wnjnz**
getting started guide, fleshed out features list more

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
-# Asys
+# ASYS VOLUNTEER LEAD
 
-Asys is a web app that helps volunteer organizations manage their front-line operations in disaster relief.
+ Asys volunteer lead is an app made by volunteers for volunteers that empowers volunteers with information to respond where they are needed the most. This helps volunteer organizations manage not only their front-line operations in disaster relief but also long-term volunteer effots. This app uses [react-native](https://facebook.github.io/react-native/) and [typescript](https://www.npmjs.com/package/typescript)
```
Comment (iAmWillShepherd) [inline README.md]: Grammar, *Asys volunteer lead* is a proper noun and should be capitalized.

Comment (iAmWillShepherd) [reply README.md]: Let's use volunteers less.

Comment (iAmWillShepherd) [reply README.md]: Grammar, *relief but also long-term* missing comma

**Review by iAmWillShepherd — COMMENTED** [review]
Some grammar, but your choice to fix or not.

**Comment by wnjnz** [comment]
fixed grammar.

**Pull request merged**